### DISCUSSION TITLE
Can I change the regional format settings into german (europa)?

### DISCUSSION THREAD

**Discussion by NemiSinc** [getredash/redash#6113] [Q&A]
Hello everyone,

I'm currently facing an issue with formatting. I noticed that the decimal digits are spaced with a dot (.) in the American standard, but in Germany, we typically use a comma (,). For example: USA 123,245,652.00 and in Germany we would write this number like this 123.245.652,00 
I would like to know if there's a way to change the regional format settings to align with the German standards.

I've explored the options within the visualization settings and found that while it allows format changes in regular visualizations, this setting is not available when working with pivot tables. Could anyone please advise me on how to address this formatting issue in pivot tables as well?

I appreciate any assistance or guidance you can provide. Thank you in advance!"

Best,
Nemi

**Comment by lemkepf** [comment]
Were finding a similar issue. Were you able to find a solution?

**Comment by lscapim** [comment]
Hi, we have the same situation here, we needed to change the decimal places, any developments on the matter? 
We need to bring division by comma in table views
